### DISCUSSION TITLE
How to custom on my data

### DISCUSSION THREAD

**Discussion by luongphambao** [UTokyo-FieldPhenomics-Lab/EasyIDP#91] [General]
Hi authors, i want to custom param on my data, you can help me?

**Comment by HowcanoeWang** [comment]
[How To Ask Questions The Smart Way#Before You Ask](https://github.com/selfteaching/How-To-Ask-Questions-The-Smart-Way/blob/master/How-To-Ask-Questions-The-Smart-Way.md#before-you-ask)

We could do nothing with the current info you provided.
